indexers/staking-squid/src/events/epoch.ts

```diff
@@ -273,8 +294,9 @@ export async function processEpochTransitionEvent(
     operatorsIds.includes(o.operatorId.toString())
   );
 
-  for (const deposit of allDeposits) {
+  for (const [index, deposit] of allDeposits.entries()) {
```

Comment:
Could be simplified to just `allDeposits.values()`?

Reply:
yeah indeed the changes was not needed, I had changed to access the index, but I reverted the code, except that line, thanks